### PR TITLE
Call it the Chef install script

### PR DESCRIPTION
**Pull request by tas50**
If you're not collecting a paycheck from Chef you probably don't care that we call it the Omnitruck script. Just call it the Chef install script.

### CODE REVIEW

**Comment by schisamo** [comment]
We'll need to update references at https://docs.chef.io/packages.html#omnitruck-install-scripts also.

**Review by kagarmoe — COMMENTED** [review]
I like this change, but to be consistent we will need to make in in 23 different documents. Should we consider changing the page & url to `install_chef` ?

File: chef_master/source/install_chef_air_gap.rst [hunk 1]
```diff
@@ -66,7 +66,7 @@ In this section you'll install the Chef server, and create your organization and
 
 .. note:: If you intend to use Chef Automate, create the ``delivery`` user and add it to your organization during this step.
 
-#. Download the package from https://downloads.chef.io/chef-server/.
+#. Download the package from https://omnitruck.chef.io/chef-server/.
```
Comment (tas50) [inline chef_master/source/install_chef_air_gap.rst]: This should be https://downloads.chef.io/chef-server/

File: chef_master/source/install_server_ha.rst [hunk 2]
```diff
@@ -454,10 +454,10 @@ The ``chef-backend-ctl gen-server-config`` command, which can be run as root fro
 Software Versions
 ----------------------------------------------------------------
 
-The backend HA cluster uses the omnibus installer (https://github.com/chef/omnibus) to package all of the software
+The backend HA cluster uses the Chef installer (https://github.com/chef/omnibus) to package all of the software
```
Comment (tas50) [inline chef_master/source/install_server_ha.rst]: We can remove that Github link

**Pull request merged**